data/test/scenarios/wml_tests/UnitsWML/AbilitiesWML/heals/heals_wfl_max_value.cfg

```diff
@@ -0,0 +1,60 @@
+# wmllint: no translatables
+
+#####
+# API(s) being tested: [heals]value=
+##
+# Actions:
+# Give the leaders a heals ability with both the value and sub attributes
+# Spawn a Mage next to the side 1 leader.
+# Set all units' hitpoints to 1.
+# Wait a turn for healing to take place.
+##
+# Expected end state:
+# max_value works.
+# The Mage has 4 hp: 1 + 2 (rest healing) + 1 ([heals])
```

Comment:
sub? And may be better representation of test to setup sides so that rest healing is not involved

Reply:
for that, Mage should make an action before turn change, it's more easy to integrate rest healing in calculation, it's already used in other test of heals/regenerate

Reply:
still talks about sub

Reply:
corercted now.